### PR TITLE
Worst and winningest coach

### PR DESCRIPTION
**Pull request by jdcarey128**
This PR: 
- adds worst and winningest coach methods 
- rearranges other methods to fit headers

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -112,6 +112,30 @@ def test_it_can_determine_team_with_worst_winning_percentage
...
+  def test_it_can_organize_season_win_percentage_for_each_team ###
+    expected = {
+      1 => 18.75,
+      4 => 31.25,
+      6 => 43.75,
+      14 => 12.5,
+      26 => 37.5
+    }
+    assert_equal expected, @stats.all_teams_win_percentage("20142015")
+  end
```
Comment (jakeheft) [inline test/stat_tracker_test.rb:129]: We need to come back to this method to ensure we are getting the proper win percentages

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -142,19 +142,6 @@ def worst_team(season)
     end.first
   end
 
-# ~~~ Game Methods ~~~
```
Comment (dunlapww) [inline lib/stat_tracker.rb:145]: Did you intend to delete these two methods?

**Review by dunlapww — COMMENTED** [review]
hi Joshua, it looks like you're deleting a couple game methods.  Do you intend to delete 'lowest_total_score' and 'highest_total_score'?

File: lib/stat_tracker.rb (hunk 2 above)
Comment (jakeheft) [reply lib/stat_tracker.rb:145]: I saw that too - they occur later in the class, so just out of order a little bit.

**Comment by jakeheft** [comment]
Everything looks good. Per our DTR, if someone else approves, please merge PR.

**Review by jakeheft — APPROVED** [review]

**Pull request merged**